pkgs/intl4x/lib/list_format.dart

```diff
@@ -2,6 +2,13 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'intl4x.dart';
+
 export 'src/list_format/list_format.dart';
 export 'src/list_format/list_format_options.dart';
 export 'src/options.dart';
+
+extension DatetimeFormatIntl4x on List<String> {
```

Comment:
Looks like a copy/paste error from the `DateTime` extension:

```suggestion
extension ListFormatIntl4x on List<String> {
```

And maybe `DatetimeFormatIntl4x` should be called `DateTimeFormatIntl4x` (with an uppercase “T”)?